test/sharness/t0183-namesys-pubsub.sh

```diff
@@ -56,15 +62,15 @@ test_expect_success 'wait for the flood' '
 
 test_expect_success 'resolve name in subscriber nodes' '
     echo "/ipfs/$HASH_FILE" > expected &&
-    ipfsi 1 name resolve /ipns/$PEERID_0 > name1 &&
-    ipfsi 2 name resolve /ipns/$PEERID_0 > name2 &&
+    ipfsi 1 name resolve /ipns/$PEERID_0_BASE36 > name1 &&
+    ipfsi 2 name resolve /ipns/$PEERID_0_BASE36 > name2 &&
```

Comment:
Unless I've missed them somewhere else we should add some sanity checks that IPNS over PubSub works (i.e. publishes + resolves) with b36, b58, and when publishing with one and resolving with the other. 

It's probably fine, but given that IPNS over PubSub is experimental it's easier for bugs to creep in and get missed

Reply:
It turns out that to do this, we need to either: (a) add --ipns-base to 'ipfs add' or (b) expand the supported bases for 'ipfs cid'.

Reply:
Why? We could just do `ipfs name resolve b58key` and `ipfs name resolve b36key` and that should just work.

Looking at this file I don't think we're really testing IPNS over PubSub very well anyway since the DHT is likely very fast in this two node test resulting in the tests passing even if PubSub is sort of broken. We don't have to fix this issue in this PR though.